lab1/heapsort.py

```diff
@@ -0,0 +1,60 @@
+import time
+
+swaps_counter = 0
+comparisons_counter = 0
+
+def heapify(arr, n, i, order):
+	largest = i
+	l = 2 * i + 1
+	r = 2 * i + 2
+	global swaps_counter
+	global comparisons_counter
+	if order == "asc":
+		comparisons_counter+=1
+		if l < n and arr[i] < arr[l]:
```

Comment:
please rename` l`, `n`, `arr` to something that might give a clue what does it do

Reply:
Renamed